src/components/Call-To-Actions/CTA_Book/index.js

```diff
@@ -0,0 +1,96 @@
+import React from "react";
```

Comment:
Nope, this is not a component. Just add it individually in a file named `cta_book.js` under the `Individual-Integrations` section folder and use it from there.

Reply:
done.